src/main/java/org/jabref/gui/groups/GroupTreeView.java

```diff
@@ -71,13 +72,15 @@
 
     private GroupTreeViewModel viewModel;
     private CustomLocalDragboard localDragboard;
+    private JabRefPreferences preferences;
 
     private DragExpansionHandler dragExpansionHandler;
 
     @FXML
     public void initialize() {
         this.localDragboard = stateManager.getLocalDragboard();
         viewModel = new GroupTreeViewModel(stateManager, dialogService, preferencesService, taskExecutor, localDragboard);
+        preferences = JabRefPreferences.getInstance();
```

Comment:
Please use the existing PreferencesService class, it is already a wrapper/it extends the JabRefPreferences class.
Just add a method there for getting the preferences key

Reply:
Thank you for comment. Just want to make sure, that I understand you correctly:
- create a class for group preferences (like "GeneralPreferences", but for groups);
- add a methods to PreferencesService interface: getGroupPreferences, storeGroupPreferences;
- implement those methods in JabRefPreferences class;
- use those methods in GroupTreeeView class to access preferences key;

Correct?

Reply:
Hi,
even easier, no need for a GroupPreferences 
1. Add a method like getDisplayGroupCount() to the PreferencesService Interface
2. Add the same method to JabRefPreferences (override annoation) and  simply return 
`return getBoolean(JabRefPreferences.DISPLAY_GROUP_COUNT)`
3. In GroupTreeView you call `preferencesService.getDisplayGroupCount()`

I think renaming quantity to Count fit's better